### PR TITLE
:book: update README.md

### PR DESCRIPTION
**Pull request by tuminoid**
Update README.md with 1.5 build badges and update compatibility table.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
+[![Ubuntu E2E Integration 1.5 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-5_e2e_integration_test_ubuntu&subject=Ubuntu%20e2e%20integration%201.4)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_release-1-5_integration_test_ubuntu/)
+[![CentOS E2E Integration 1.5 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-5_e2e_integration_test_centos&subject=Centos%20e2e%20integration%201.4)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_release-1-5_integration_test_centos/)
+[![Ubuntu E2E feature 1.5 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-5_e2e_feature_test_ubuntu/&subject=Ubuntu%20E2E%20feature%201.4)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_release-1-5_e2e_feature_test_ubuntu/)
+[![CentOS E2E feature 1.5 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-5_e2e_feature_test_centos/&subject=CentOS%20E2E%20feature%201.4)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_release-1-5_e2e_feature_test_centos/)
```
Comment (lentzi90) [inline README.md:6]:
```suggestion
[![Ubuntu E2E Integration 1.5 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-5_e2e_integration_test_ubuntu&subject=Ubuntu%20e2e%20integration%201.5)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_release-1-5_integration_test_ubuntu/)
[![CentOS E2E Integration 1.5 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-5_e2e_integration_test_centos&subject=Centos%20e2e%20integration%201.5)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_release-1-5_integration_test_centos/)
[![Ubuntu E2E feature 1.5 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-5_e2e_feature_test_ubuntu/&subject=Ubuntu%20E2E%20feature%201.5)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_release-1-5_e2e_feature_test_ubuntu/)
[![CentOS E2E feature 1.5 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-5_e2e_feature_test_centos/&subject=CentOS%20E2E%20feature%201.5)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_release-1-5_e2e_feature_test_centos/)
```

Comment (tuminoid) [reply README.md]: Fixed, good catch.

**Review by lentzi90 — COMMENTED** [review]
/approve

**Review by kashifest — COMMENTED** [review]
/lgtm

**Pull request merged**